docs/source/short_docs.rst

```diff
@@ -20,6 +20,7 @@
 * :ref:`is-managed`       -- Check if the current branch is managed by git machete (mostly for scripts)
 * :ref:`list`             -- List all branches that fall into one of pre-defined categories (mostly for internal use)
 * :ref:`log`              -- Log the part of history specific to the given branch
+* :ref:`prune`            -- Delete local branches that have been deleted remotely
```

Comment:
There's been even `prune-branches` a long time ago (v1.x), later removed in favor of `delete-unmanaged` in the 1.x->2.0 version bump, as more precise 😅 so yeah, indeed I'd rather **not** use this name

Reply:
suggestions? `delete-merged`?